src/install.sh

```diff
@@ -1,8 +1,7 @@
-#!/bin/bash -e
```

Comment:
are you sure you want to remove this? the script will continue if there's a failure on 1 command. It could leave it in a bad state

Reply:
I set it further down, AFTER I test xargs for option values, where I don't want it to exit on failure (because I'm testing for failure)